README.md

```diff
@@ -1,6 +1,7 @@
 # Azure Function Enforcer Example
 
 See the full official documentation for the Human Security Azure Enforcer [here](https://docs.perimeterx.com/docs/installation-azure).
+You can use the `enforcer-setup.cmd1 file to automate the enforcer setup process, or you can do it manually by following the instructions:
```

Comment:
File will be titled`enforcer-setup.cmd` according to PR #6 

I would also include other information about the limitations of this script (e.g., only for Windows, no custom first party routes, no custom filter by routes).

Reply:
Also, why not add this change to #6 ? We want to merge both together anyway.